spec/controllers/api/coordinates_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+require 'geocoding'
+
+RSpec.describe Api::CoordinatesController, type: :controller do
+  let(:json) { JSON.parse(response.body, symbolize_names: true) }
+  let!(:location_query) { 'ct9' }
+
+  before(:each, :json) do
+    request.accept = 'application/json'
+  end
+
+  before(:each) do
+    request.headers['origin'] = Rails.application.config.allowed_cors_origin
+  end
+
+  describe 'GET /api/v1/coordinates/ct9.html' do
+    it 'returns status :not_found as only CSV and JSON format is allowed' do
+      get :show, params: { api_version: 1, location: location_query }, format: :html
+
+      expect(response).to have_http_status(:not_found)
+    end
+  end
+
+  describe 'GET /api/v1/coordinates/ct9.json', json: true do
+    let(:geocoding) { Geocoding.new(location_query) }
```

Comment:
I'm implementing [this](https://github.com/alexreisner/geocoder#testing) in `rails_helper` as part of the search work in order to mock all Geocoding requests in testing by default

Reply:
Looks good. When you have updated `rails_helper` on master I will use it here.